src/Apps/Artist/Routes/AuctionResults/Components/AuctionResultHeader.tsx

```diff
@@ -0,0 +1,11 @@
+import { Box, Sans, Serif } from "@artsy/palette"
+import React from "react"
+
+export const AuctionResultHeader = () => (
```

Comment:
Minor nit -- can the following components (here and below) use explicit `return` statements? When this code needs to be returned to for debugging, the addition of intermediate variables, logs, etc, implicit returns will have to be unwound. (Code is aways returned to given enough time because code is never technically "complete", and implicit returns on react components seem to imply that further changes will not be necessary.)

Reply:
Yes I keep forgetting this. Can we enforce it with lint/prettier?

Reply:
It's not amenable to a linter or prettier, because implicit returns make sense in lots of different contexts, just not in React code that spans multiple lines. It's rare that one writes a multi-line function without a return statement; React components are just regular functions.

Reply:
wondering if the following linter rule is possible:

- is a method
- which returns JSX
- which spans more than 1 line
- which has no `return`

then can warn/correct